src/nctIndex.jsx

```diff
@@ -94,6 +95,7 @@ async function init() {
           <BrowserRouter basename={basename}>
             <div className='main-page'>
               {GA.init(gaTracking, dev, gaDebug) && <RouteTracker />}
+              {injectDAPTag && <DAPRouteTracker />}
```

Comment:
as mentioned on slack, i would make the URLs configurable and add the `DAPRouteTracker` in all index pages instead of only the NCT one so that it works for all Commons